### PR TITLE
Move showdown extensions to ghost-showdown

### PR DESCRIPTION
**Pull request by ErisDS**
no issue
- We already maintain our own fork of showdown, this moves our custom extensions to our fork
- Code duplication is removed
- Tests are also moved to the other repo

### CODE REVIEW

File: core/server/models/post.js [hunk 1]
```diff
@@ -5,10 +5,7 @@ var _              = require('lodash'),
     sequence       = require('../utils/sequence'),
     errors         = require('../errors'),
     Showdown       = require('showdown-ghost'),
-    ghostgfm       = require('../../shared/lib/showdown/extensions/ghostgfm'),
-    footnotes      = require('../../shared/lib/showdown/extensions/ghostfootnotes'),
-    highlight      = require('../../shared/lib/showdown/extensions/ghosthighlight'),
-    converter      = new Showdown.converter({extensions: [ghostgfm, footnotes, highlight]}),
+    converter      = new Showdown.converter({extensions: ['ghostgfm', 'footnotes', 'highlight']}),
```
Comment (novaugust) [inline core/server/models/post.js]: Do the extensions still get written in this way?

**Comment by novaugust** [comment]
Much :+1:, restarted failed build

File: core/server/models/post.js (hunk 1 above)
Comment (ErisDS) [inline core/server/models/post.js]: All 4 extensions are now part of showdown proper, so yep :) 
Ref: https://github.com/showdownjs/showdown#extensions

**Comment by ErisDS** [comment]
:grin:

**Pull request merged**